docs/site/index/survey-tool/managing-users.md

```diff
@@ -112,17 +127,16 @@ Later on, in the vetting phase, you can send messages with the outstanding dispu
 10. ~~Go get a cup of coffee (this will take a little while)~~
 11. ~~At the bottom of the screen, you'll see a list of who has participated, and how many submits and votes they've each made.~~
 
-## To check if you have "bad" locales (locales assigned that aren't in CLDR)
-
-- 1. Go to Coverage (up at the top right in the survey tool)
-	2. If you see the following message, then some of your users have bogus locales OR locales that aren't in CLDR.
-	3. **Warning: locale designations not matching locales in CLDR: X....**
-	4. Go to Manage Users (top right), and look at the users signed up to these locales
-	5. When they are bogus fix them (work with translator to replace by real locale ids)
-	6. When they are valid, you may propose to the committee that we add a locale:
-		1. Valid ones are under languages in en.xml
-		2. OR (unusual case) under languages in http://www.ietf.org/internet\-drafts/draft\-ietf\-ltru\-4645bis\-04\.txt
-		3. However, watch for mistakes, eg someone using "tw" to mean Taiwanese, when it actually means "Twi"
+## NEEDS WORK: To check if you have "bad" locales (locales assigned that aren't in CLDR)
+
+1. Go to Coverage (up at the top right in the survey tool).
+2. If you see the following message, then some of your users have bogus locales OR locales that aren't in CLDR: **Warning: locale designations not matching locales in CLDR: X....**
```

Comment:
This also needs a ticket to disallow bogus/non-CLDR locales.

Reply:
They are disallowed from voting in bogus/non-CLDR locales, and disallowed from creating users with such.  Do you mean to remove this warning from the UI? That is, suppose `iw` was renamed `he` (hey, it could happen), what happens to users assigned to `iw`? This warning says, you need to go in and choose a different locale for them. We could have no warning, and then there would be no indication that something was inconsistent.

So, please clarify 'disallow'.

Reply:
If we reversed an alias, then we would have to make substantial changes to both data on disk **and** fix the ST database.

So Managers should **never** see that message, and we should drop any discussion of it instead of taking up their time trying to understand what to do in a case that won't happen.

Reply:
fully agree for this discussion.

Reply:
In my later draft I removed this entire section because I couldn't find this interface in the survey tool. There is no "Coverage" link that I could find.